src/SFA.DAS.ApprenticeFeedback.Domain/Models/FeedbackTransaction.cs

```diff
@@ -0,0 +1,40 @@
+﻿using SFA.DAS.ApprenticeFeedback.Domain.Configuration;
+using SFA.DAS.ApprenticeFeedback.Domain.Interfaces;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using static SFA.DAS.ApprenticeFeedback.Domain.Models.Enums;
+
+namespace SFA.DAS.ApprenticeFeedback.Domain.Models
+{
+    public class FeedbackTransaction
+    {
+        public Guid Id { get; set; }
+        public Guid? ApprenticeFeedbackTargetId { get; set; }
+        public string EmailAddress { get; set; }
+        public string FirstName { get; set; }
+        public Guid? TemplateId { get; set; }
+        public DateTime CreatedOn { get; set; }
+        public DateTime? SendAfter { get; set; }
+        public DateTime? SentDate { get; set; }
+
+
+        public static implicit operator FeedbackTransaction(Entities.FeedbackTransaction source)
```

Comment:
Are there unit tests to check the mapping?

Reply:
Added mapping unit tests